tools/bpf2c/templates/kernel_mode_bpf2c.vcxproj

```diff
@@ -40,6 +40,9 @@
     <RootNamespace>$(FileName)</RootNamespace>
     <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
     <ProjectName>$(FileName)_km</ProjectName>
+    <!-- Required for Microsoft OneBranch builds -->
+    <UseInternalMSUniCrtPackage>true</UseInternalMSUniCrtPackage>    
```

Comment:
question: does this allow linking with `ucrtbase.dll` which is shipped inbox with Windows?

Reply:
I think it does, but let's follow up on that separate from this PR.